modes/microscopy/public/assets/icons/tool-circle.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32">
```

Comment:
where do we inject these icons that are inside your mode?

Reply:
Removed those icons, as we have added the same icons into '/platform/viewer'